### PR TITLE
Have added project skeleton

### PR DESCRIPTION
**Pull request by AndriiHliuza**

### CODE REVIEW

File: .mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (SergeyKostyniuk) [inline .mvn/wrapper/maven-wrapper.properties:1]: Looks that this should be ignored

File: mvnw [hunk 2]
```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
```
Comment (SergeyKostyniuk) [inline mvnw:2]: This file also should be ignored

**Review by Serhii-Kostyniuk — APPROVED** [review]
Approve, but with two comments, please fix before merging

File: mvnw (hunk 2 above)
Comment (AndriiHliuza) [reply mvnw]: Removed those files

**Pull request merged**